pkg/settings/settings_test.go

```diff
@@ -211,6 +211,45 @@ func TestSettings_IsOutputFormatCamelCase(t *testing.T) {
 	}
 }
 
+func TestSettings_IsFileNameCasingSnakeCase(t *testing.T) {
+	tests := []struct {
+		desc     string
+		settings func() *Settings
+		expected bool
+	}{
+		{
+			desc:     "in default settings camel case will be used",
+			settings: New,
+			expected: false,
+		},
+		{
+			desc: "use snake case",
+			settings: func() *Settings {
+				s := New()
+				s.FileNameCasing = FileNameCasingSnakeCase
+				return s
+			},
+			expected: true,
+		},
+		{
+			desc: "any other output format will converted to camel case",
+			settings: func() *Settings {
+				s := New()
+				s.FileNameCasing = FileNameCasing("any")
+				return s
+			},
+			expected: false,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.desc, func(t *testing.T) {
+			settings := test.settings()
+			actual := settings.IsFileNameCasingSnakeCase()
+			assert.Equal(t, test.expected, actual)
+		})
+	}
+}
+
```

Comment:
Can you also write unit tests for the FileNameFormat's Set method?

Reply:
Yes